### DISCUSSION TITLE
Code for sending email, which or what, and where to insert it

### DISCUSSION THREAD

**Discussion by giusebos** [joomengine/Joomla-Component-Builder#886] [General]
I would like to create a contact form, what is the part of the code to insert for sending?
Is there a tutorial for this? Where can I find information?

**Comment by Safweb** [comment]
https://www.google.com/search?client=firefox-b-d&q=code+contact+form

**Comment by Llewellynvdm** [comment]
In the **component area** under the `Libs & Helpers` tab there is a button:
![image](https://user-images.githubusercontent.com/5607939/161558456-0797a2fb-b71e-46f4-be99-ce859c12d19e.png)

This will add a helper class to your component that integrates with the Joomla email API/class.

To use this helper class look at these tutorial:

- [How to use email helper in your components](https://www.youtube.com/watch?v=tp6mMUTOF2Y&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=25)
- [How to setup a store message method along side the email helper class](https://www.youtube.com/watch?v=peVNLsAncGY&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=26)

**Reply by giusebos** [reply]
it would be very useful to have an example component, also not to copy all that code, without making mistakes.

I have seen the 2 video tutorials, but I don't understand which of the 2 to use, what is the difference ...

Grazie!
